pkgs/tools/filesystems/pfsshell/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2020-01-10";
```

Comment:
What is the reason to do not ship the stable release?

Reply:
Mainly because they' ve changed the buildsystem to meson and i wouldn't rewrite the derivation twice.